### DISCUSSION TITLE
Component Tabs was not imported, exported, or provided by MDXProvider as global scope

### DISCUSSION THREAD

**Discussion by peterroe** [facebook/docusaurus#6034] [Q&A] [answered]
I upgrade docusaurus from @2.0.0-beta.7 to @2.0.0-beta.9, and I find `Tabs` component cant's used, I remember it will import autolly in @2.0.0-beta.7? 
cant's work now:
```js
import Tabs from '@theme/Tabs'
import TabItem from '@theme/TabItem'
```

**Comment by Josh-Cena** [comment, marked as answer]
I'm having the same warning messages. However I believe the build output is still normal?

**Reply by peterroe** [reply]
I see. you are right. When I saw the warning, I thought it was the reason for the failure. I found that I couldn't build it for other reasons. thank you help.

**Reply by Josh-Cena** [reply]
Yes, FYI the Docusaurus production build is also reporting this warning... We'll investigate later since nothing's actually failing

**Comment by mcclowes** [comment]
Still happens for me

**Reply by slorber** [reply]
Repro or it doesn't happen. Use docusaurus.new
